apps/bfd-server/bfd-server-launcher/src/test/java/gov/cms/bfd/server/launcher/DataServerLauncherAppIT.java

```diff
@@ -29,6 +33,32 @@
 public final class DataServerLauncherAppIT {
   /** The POSIX signal number for the <code>SIGTERM</code> signal. */
   private static final int SIGTERM = 15;
+  /** Regex for access log entries */
+  private static final String accessLogPattern =
+      "^(\\S+) \\S+ \\\"([^\\\"]*)\\\" \\[([^\\]]+)\\] \\\"([A-Z]+) ([^ \\\"]+) HTTP\\/[0-9.]+\\\" \\\"([^ \\\"]+)\\\" ([0-9]{3}) ([0-9]+|-) ([0-9]+|-) (\\S+) ([0-9]+|-) \\[([^\\]]+)\\] ([0-9]+|-) \\\"([^\\\"]*)\\\" ([0-9]+|-) \\\"([^\\\"]*)\\\" ([0-9]+|-) \\\"([^\\\"]*)\\\" (\\S+)";
```

Comment:
wondering if you can break this up a little bit so that the pieces are easier to think about individually. perhaps like this:

```java
private static final String accessLogPattern =
   "^(\\S+) " + \\ Description of first field
   "\\S+ " + \\ Description of second field
   ...
```

Stringbuilder would be an option as well using append() for each piece.

If you come up with something nice you could also use it below for the positive and negative tests.